tests/testthat/test-compile_report.R

```diff
@@ -0,0 +1,175 @@
+library(fs)
+
+test_that("test parameteriesed report output", {
```

Comment:
typo; should this file be named `test-compile_reports`?

Reply:
closed with https://github.com/reconhub/reportfactory/pull/68/commits/e60198f05faaf899a33aab96f3d9634ab3413252